packages/tutanota-crypto/test/RsaTest.ts

```diff
@@ -18,9 +18,10 @@ import {
 	rsaPrivateKeyToHex,
 	rsaPublicKeyToHex,
 } from "../lib/encryption/Rsa.js"
-import { parseBigInt } from "../lib/internal/crypto-jsbn-2012-08-09_1.js"
 import { SecureRandom } from "../lib/random/SecureRandom.js"
 import { CryptoError } from "../lib/error.js"
+import { parseBigInt } from "../lib/internal/crypto-jsbn-2012-08-09_1.js"
+import crypto from "node:crypto"
```

Comment:
in this test we use `globalThis.crypto` (which we also might stub), we should avoid importing node modules